netbench/cdk/src/main/java/com/aws/ReportStack.java

```diff
@@ -0,0 +1,32 @@
+package com.aws;
+
+import software.constructs.Construct;
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+import software.amazon.awscdk.services.s3.Bucket;
+import software.amazon.awscdk.RemovalPolicy;
+import software.amazon.awscdk.Environment;
+
+
+public class ReportStack extends Stack {
+    private Bucket metricsBucket;
+
+    public ReportStack(final Construct parent, final String id) {
+        this(parent, id, null);
+    }
+
+    public ReportStack(final Construct parent, final String id, final StackProps props) {
+        super(parent, id, props);
+
+        //Environment env = props.getEnv();
+        metricsBucket = Bucket.Builder.create(this, "MetricsReportBucket")
```

Comment:
Since S3 Buckets have a global namespace I would suggest to at least include the following parameterized variables in the name:

* Region
* AWS Account

It could be useful to partition by protocol as well but I don't think that is important.

Reply:
I'm currently having CDK automatically generate a unique bucket name for me. However, I could provide a more descriptive name with the above attributes if you think that would be more clear for users.

Reply:
Oh I didn't realize. I think what you have now is good. In the future for scripts / creating dashboards it may be useful to be able to predict the bucket name but I think this is a two way door

Reply:
Okay, great, thanks for the feedback